cpp/tests/copying/slice_tests.cpp

```diff
@@ -344,3 +344,20 @@ TEST_F(SliceTableCornerCases, NegativeOffset)
 
   EXPECT_THROW(cudf::slice(src_table, indices), cudf::logic_error);
 }
+
+TEST_F(SliceTableCornerCases, MiscOffset)
+{
+  cudf::test::fixed_width_column_wrapper<int32_t> col2{
```

Comment:
Just to verify :  the "begin bit" here will end up being within the first word because we're splitting at index 19 (ie < 32)?

Reply:
Yes, you are correct.